src/views/ModelPlan/TaskList/Payment/ClaimsBasedPayment/index.tsx

```diff
@@ -0,0 +1,577 @@
+                                }
+                              </FieldErrorMsg>
+                              <Field
+                                as={TextAreaField}
+                                className="height-15"
+                                error={
+                                  flatErrors.affectsMedicareSecondaryPayerClaimsHow
+                                }
+                                id="payment-affects-medicare-secondary-payer-claims-how"
+                                data-testid="payment-affects-medicare-secondary-payer-claims-how"
+                                name="payment-affects-medicare-secondary-payer-claims-how"
+                              />
+                            </FieldGroup>
+                          )}
+                          <Field
+                            as={Radio}
+                            id="payment-affects-medicare-secondary-payer-claims-No"
+                            name="payment-affects-medicare-secondary-payer-claims"
+                            label={h('no')}
+                            value="FALSE"
+                            checked={
+                              values.affectsMedicareSecondaryPayerClaims ===
+                              false
+                            }
+                            onChange={() => {
+                              setFieldValue(
+                                'affectsMedicareSecondaryPayerClaims',
+                                false
+                              );
+                            }}
+                          />
+                        </Fieldset>
+                        <AddNote
+                          id="payment-affects-medicare-secondary-payer-claims-note"
+                          field="affectsMedicareSecondaryPayerClaimsNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="payment-affect-current-policy"
+                        error={!!flatErrors.payModelDifferentiation}
+                        className="margin-top-4"
+                      >
+                        <Label
+                          htmlFor="payment-affect-current-policy"
+                          className="maxw-none"
+                        >
+                          {t('affectCurrentPolicy')}
+                        </Label>
+                        <FieldErrorMsg>
+                          {flatErrors.payModelDifferentiation}
+                        </FieldErrorMsg>
+                        <Field
+                          as={TextAreaField}
+                          className="height-15"
+                          error={flatErrors.payModelDifferentiation}
+                          id="payment-affect-current-policy"
+                          data-testid="payment-affect-current-policy"
+                          name="payment-affect-current-policy"
```

Comment:
Same as above, the name should reflect the field name, Formik needs it to be exact